helix-term/src/commands/lsp.rs

```diff
@@ -1424,3 +1424,88 @@ fn compute_inlay_hints_for_view(
 
     Some(callback)
 }
+
+pub fn pull_diagnostic_for_current_doc(editor: &Editor, jobs: &mut crate::job::Jobs) {
+    let doc = doc!(editor);
+    let Some(language_server) = doc
+        .language_servers_with_feature(LanguageServerFeature::PullDiagnostics)
+        .next()
```

Comment:
It looks like this is only requesting diagnostics from the first language server to support it. We should request pull diagnostics for all language servers in this iterator - there might be multiple language servers configured that support the feature.

Reply:
Yes that was not intentional

Reply:
I have a hard time testing if multiple servers work, however it is working with the one language server that i am using.